.gitignore

```diff
@@ -14,6 +14,12 @@ sdks/**/vendor/**/*
 runners/**/vendor/**/*
 **/.gradletasknamecache
 
+# Ignore gradle files generated by Intellij
+**/gradle-wrapper.jar
```

Comment:
Does IntelliJ add new wrapper files, or update existing ones? If it's adding new files, it would be nice if we could craft the exclusion here to only apply to the IntelliJ subdirectory and not the actual gradle wrappers.

Reply:
It updates our existing ones.